Test Automation Course/src/abstraction/Patrat.java

```diff
@@ -0,0 +1,47 @@
+package abstraction;
+
+
+import java.text.DecimalFormat;
+
+public class Patrat extends FormaGeometrica {
```

Comment:
as sugera sa redenumesti aceasta clasa in patrulater, la patrat toate laturile sunt egale :)
si deja daca cumva vezi ca toate sunt egale - sa construiesti un obiect de tip patrat care are doar lungimea la o latura, poti sa mostenesti de ex 3 clase de la patrulater, Patrat, dreptunghi si romb, va fi exemplu bun de mostenire de mai multe nivele

Reply:
Se poate de facut, doar ca necesita mai mult timp, acum nu reusesc :( , cind voi avea timp liber ma voi juca cu mostenirea si  voi crea mai multe clase. Pina cind am rezolvat toate comentariile de mai jos si am facut sarcina cu exceptiile. Poti verifica totul in ultimul commit. Mersi !!!